dev/RELEASE_PROCESS.rst

```diff
@@ -3,40 +3,64 @@ Release Process
 
 This document describes the process for creating new releases.
 
-1. First, you should build wheels that you'd like to use for testing. That can
-   be done by following the `documentation for building wheels`_.
+1. **Increment the Python version:** Create a PR that increments the Python
+   package version. See `this example`_.
 
-2. **Testing:** Before a release is created, significant testing should be done.
+2. **Download the Travis-built wheels:** Once Travis has completed the tests,
+   the wheels from this commit can be downloaded from S3 to do testing, etc.
+   The URL is structured like this:
+   ``https://s3-us-west-2.amazonaws.com/ray-wheels/<hash>/ray-<version>-cp27-cp27mu-manylinux1_x86_64.whl``
+   where ``<hash>`` is replaced by the SHA of the commit and the ``<version>``
+   is the incremented version from the previous step.
+
+3. **Create a release branch:** This branch should also have the same SHA as the
+   previous two steps. In order to create the branch, locally checkout the SHA
+   i.e. ``git checkout <hash>``. Then checkout a new branch of the format
+   ``releases/<release_number>``. The release number must match the increment in
+   the first step. Then push that branch to the ray repo:
+   ``git push upstream releases/<release_number>``.
+
+4. **Testing:** Before a release is created, significant testing should be done.
    Run the script `ci/stress_tests/run_stress_tests.sh`_ and
    `ci/stress_tests/run_application_stress_tests.sh`_ and make sure it
-   passes. *And make sure it is testing the right version of Ray!* This will use
-   the autoscaler to start a bunch of machines and run some tests. Any new
-   stress tests should be added to this script so that they will be run
+   passes. *And make sure it is testing the wheels created in step 2!* This
+   will use the autoscaler to start a bunch of machines and run some tests.
```

Comment:
Can you actually say that they MUST modify the file autoscaler config file to test the relevant wheels from above.

And maybe modify the autoscaler config file so that it doesn't run out of the box but instead requires the correct commit ID to be inserted.

Do you know what I mean by that?